axelrod/load_data_.py

```diff
@@ -0,0 +1,45 @@
+import pkg_resources
+
+
+def load_file(filename, directory):
```

Comment:
Cool. I think anyone that uses anaconda or can pip install numpy should have access to rest of the scientific stack (for sure at least scipy and pandas).